spec/domain/queries/find_series_spec.rb

```diff
@@ -1,108 +1,93 @@
-      edition2 = create(:edition, date: day0, base_path: '/path2')
-
-      metric1 = create :metric, edition: edition1, date: day0
-      metric2 = create :metric, edition: edition1, date: day1
-      metric3 = create :metric, edition: edition1, date: day2
-      create :metric, edition: edition2, date: day1
-      create :metric, edition: edition2, date: day2
-
-      results = described_class.new.by_base_path('/path1').run
-
-      expect(results).to match_array([metric1, metric2, metric3])
-    end
-
-    it 'ignores parameters when blank' do
-      metric = create(:metric, date: Date.today)
-
-      expect(described_class.new.by_base_path('').run).to match_array([metric])
+      edition1 = create :edition, date: '2018-1-12', base_path: '/path1'
+      edition2 = create :edition, date: '2018-1-12', base_path: '/path2'
+
+      create :metric, edition: edition1, date: '2018-1-12', pviews: 1
+      create :metric, edition: edition1, date: '2018-1-13', pviews: 2
+      create :metric, edition: edition1, date: '2018-1-14', pviews: 3
+      create :metric, edition: edition2, date: '2018-1-13', pviews: 4
+      create :metric, edition: edition2, date: '2018-1-14', pviews: 5
+
+      result = described_class.new.by_metrics(%w(pviews)).by_base_path('/path1').run
+
+      expect(result.first.time_series).to eq([
+        { date: "2018-01-12", value: 1 },
+        { date: "2018-01-13", value: 2 },
+        { date: "2018-01-14", value: 3 },
+      ])
     end
   end
 
   describe '#editions' do
     it 'return the content items included in the report' do
-      day0 = Date.new(2018, 1, 12)
-      day1 = Date.new(2018, 1, 13)
-      day2 = Date.new(2018, 1, 14)
-      edition1 = create(:edition, base_path: '/path1', date: day0)
-      edition2 = create(:edition, base_path: '/path2', date: day0)
-
-      create(:metric, edition: edition1, date: day0)
-      create(:metric, edition: edition1, date: day1)
-      create(:metric, edition: edition1, date: day2)
-      create(:metric, edition: edition2, date: day1)
-      create(:metric, edition: edition2, date: day2)
-
-      expect(described_class.new.editions).to match_array([edition1, edition2])
+      edition1 = create :edition, base_path: '/path1', date: '2018-1-12'
+      edition2 = create :edition, base_path: '/path2', date: '2018-1-12'
+
+      create :metric, edition: edition1, date: '2018-1-12', pviews: 1
+      create :metric, edition: edition1, date: '2018-1-13', pviews: 2
+      create :metric, edition: edition1, date: '2018-1-14', pviews: 4
+      create :metric, edition: edition2, date: '2018-1-13', pviews: 5
+      create :metric, edition: edition2, date: '2018-1-14', pviews: 6
+
+      result = described_class.new.by_metrics(%w(pviews)).by_base_path('/path1').run
+      expect(result.first.time_series).to eq([
+        { date: "2018-01-12", value: 1 },
+        { date: "2018-01-13", value: 2 },
+        { date: "2018-01-14", value: 4 },
+      ])
```

Comment:
Not sure how this test is different from the one previous?

Reply:
Addressed in https://github.com/alphagov/content-performance-manager/pull/959/commits/2861f50b1e9093caf4fc69a58042c88ab8653087